06/hw_dice.py

```diff
@@ -0,0 +1,27 @@
+import random
+import numpy as np
+
+
+# task 8
+def roll_dice():
+    return random.randint(1, 6)
+
+
+def roll_until_six():
+    count = 1
+    while roll_dice() != 6:
+        count += 1
+    return count
+
+
+def game():
+    gamers = []
+    for i in range(4):
+        gamers.append(roll_until_six())
+    print('Count of rolls: ', gamers)
+    winner = np.where(gamers == np.amax(gamers))[0][0] + 1
```

Comment:
Numpy je super knihovna ale tady to by šlo i bez ni, pomocí standardních operaci s listem. Věděla by jsi jak?

Reply:
zmeneno :)